### PR TITLE
New Squid and GridFTP server for UMD FD#67627

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: topology/University of Maryland/UMD-CMS/umd-cms.yaml [hunk 1]
```diff
@@ -124,11 +150,33 @@ Resources:
...
+          Name: Shabnam Jabeen
+        Secondary:
+          ID: 4cee0f130d367c26289f9f5e2ba2ca56b7b997dd
+          Name: Anwar Bhatti
+      Security Contact:
+        Primary:
+          ID: bdd821fe2ff3d859dc96f79e5f8180a7ef529d70
+          Name: Shabnam Jabeen
+    Description: squid service for frontier and cvmfs
+    FQDN: hepcms-squid.umd.edu
```
Comment (matyasselmeci) [inline topology/University of Maryland/UMD-CMS/umd-cms.yaml:175]:
```suggestion
    FQDN: hepcms-squid2.umd.edu
```

File: topology/University of Maryland/UMD-CMS/umd-cms.yaml [hunk 2]
```diff
@@ -124,11 +150,33 @@ Resources:
           ID: bdd821fe2ff3d859dc96f79e5f8180a7ef529d70
           Name: Shabnam Jabeen
     Description: squid service for frontier and cvmfs
-    FQDN: hepcms-squid.umd.edu
+    FQDN: hepcms-squid2.umd.edu
```
Comment (matyasselmeci) [inline topology/University of Maryland/UMD-CMS/umd-cms.yaml:153]:
```suggestion
    FQDN: hepcms-squid.umd.edu
```

**Review by matyasselmeci — COMMENTED** [review]
Did the FQDNs get swapped for the squids?

**Comment by timtheisen** [comment]
You're right

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**